hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmVolumeArgs.java

```diff
@@ -188,6 +237,29 @@ public int hashCode() {
     private long quotaInBytes;
     private Map<String, String> metadata;
     private OmOzoneAclMap aclMap;
+    private long objectID;
+    private long updateID;
+
+    /**
+     * Sets the Object ID for this Object.
+     * Object ID are unique and immutable identifier for each object in the
+     * System.
+     * @param objectID - long
+     */
+    public void setObjectID(long objectID) {
+      this.objectID = objectID;
```

Comment:
Just a question, why we need to have objectID and updateID?

Reply:
Object ID represents a unique Identity for an object.  Update ID represents a token that is useful to detect if an object has been changed after a read has been performed. For example, with update IDs I can cache an object and make changes, and when I apply I can detect if an object has been updated by any other thread.